### PR TITLE
fixes continuation indent in ktfmt default formatter

### PR DESCRIPTION
**Pull request by lonnelars**
Please **DO NOT FORCE PUSH**. Don't worry about messy history, it's easier to do code review if we can tell what happened after the review, and force pushing breaks that.

Please make sure that your [PR allows edits from maintainers](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/).  Sometimes its faster for us to just fix something than it is to describe how to fix it.

Allow edits from maintainers

After creating the PR, please add a commit that adds a bullet-point under the `[Unreleased]` section of [CHANGES.md](https://github.com/diffplug/spotless/blob/main/CHANGES.md), [plugin-gradle/CHANGES.md](https://github.com/diffplug/spotless/blob/main/plugin-gradle/CHANGES.md), and [plugin-maven/CHANGES.md](https://github.com/diffplug/spotless/blob/main/plugin-maven/CHANGES.md) which includes:

- [x] a summary of the change
- either
    - [ ] a link to the issue you are resolving (for small changes)
    - [x] a link to the PR you just created (for big changes likely to have discussion)

If your change only affects a build plugin, and not the lib, then you only need to update the `plugin-foo/CHANGES.md` for that plugin.

If your change affects lib in an end-user-visible way (fixing a bug, updating a version) then you need to update `CHANGES.md` for both the lib and all build plugins.  Users of a build plugin shouldn't have to refer to lib to see changes that affect them.

This makes it easier for the maintainers to quickly release your changes :)

### CODE REVIEW

**Comment by lonnelars** [comment]
I noticed that the ktfmt plugin in IntelliJ IDEA and the spotless plugin produced different results. Turns out that spotless uses the block indent instead of the continuation indent when the formatter is configured with default settings (`<ktfmt />`). This PR demonstrates the problem with a test, and fixes the bug. 

The following configuration is a workaround until this bug gets fixed:

```xml
<configuration>
    <kotlin>
        <ktfmt>
            <style>DEFAULT</style>
            <continuationIndent>4</continuationIndent>
        </ktfmt>
    </kotlin>
    <pom/>
</configuration>
```

File: plugin-maven/CHANGES.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 We adhere to the [keepachangelog](https://keepachangelog.com/en/1.0.0/) format (starting after version `1.27.0`).
 
 ## [Unreleased]
+### Fixed
+* Maven plugin with `ktfmt` default style uses correct continuation indent ([#1562](https://github.com/diffplug/spotless/pull/1562))
```
Comment (nedtwigg) [inline plugin-maven/CHANGES.md:7]: This entry should also be in `plugin-gradle/CHANGES.md` and `/CHANGES.md`

**Review by nedtwigg — CHANGES_REQUESTED** [review]
Thanks! Is missing a couple changelog entries then it can get merged.

File: plugin-maven/CHANGES.md (hunk 1 above)
Comment (lonnelars) [reply plugin-maven/CHANGES.md:7]: Yes, I added them now.

**Comment by lonnelars** [comment]
I didn't understand the ratcheting feature, that's why I have had some trouble with the formatting. I think it is fixed now, though.

**Review by nedtwigg — APPROVED** [review]
Great, thanks for the fix!

**Comment by nedtwigg** [comment]
Released in `plugin-gradle 6.15.0` and `plugin-maven 2.33.0`.

**Pull request merged**